### PR TITLE
Drop SensioFrameworkExtraBundle dependency

### PR DESCRIPTION
**Pull request by mtarld**
- Drop SensioFrameworkExtraBundle dependency
- Don't abstract anymore the PSR7/17 implementation and require `nyholm/psr7` implementation. Therefore drop `laminas/laminas-diactoros` implementation in tests
- Fix test kernel location (`Tests -> tests`)

### CODE REVIEW

File: src/Resources/config/services.xml [hunk 1]
```diff
@@ -177,5 +183,17 @@
...
+        <service id="nyholm.psr7.psr17_factory" class="Nyholm\Psr7\Factory\Psr17Factory" />
+
+        <service id="http_message_factory" class="Symfony\Bridge\PsrHttpMessage\Factory\PsrHttpFactory">
+            <argument type="service" id="nyholm.psr7.psr17_factory" />
+            <argument type="service" id="nyholm.psr7.psr17_factory" />
+            <argument type="service" id="nyholm.psr7.psr17_factory" />
+            <argument type="service" id="nyholm.psr7.psr17_factory" />
+        </service>
+
+        <service id="http_foundation_factory" class="Symfony\Bridge\PsrHttpMessage\Factory\HttpFoundationFactory" />
```
Comment (mtarld) [inline src/Resources/config/services.xml:197]: I'm not sure about these namings. I'll take any advice

**Comment by mtarld** [comment]
Static analysis failures aren't related.

File: src/Resources/config/services.xml (hunk 1 above)
Comment (chalasr) [reply src/Resources/config/services.xml:197]: let's prefix them by `league.oauth2_server.`

**Review by chalasr — APPROVED** [review]

**Comment by chalasr** [comment]
Thank you @mtarld.

**Comment by chalasr** [comment]
PR welcome for the broken CI build :)

**Pull request merged**